lib/write-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+function uniqueFile(file) {
+  var testPath = file;
+  var counter = null;
```

Comment:
Could this be `counter = 0` and you can remove the `counter || 0` check below (i.e. `counter++`)?